mmv1/third_party/terraform/services/filestore/resource_filestore_instance_test.go.tmpl

```diff
@@ -592,4 +594,116 @@ resource "google_filestore_instance" "instance" {
 }
 `, name, location, tier)
 }
-{{- end }}
\ No newline at end of file
+
+{{- end }}
+{{- if ne $.TargetVersionName "ga" }}
+
+func TestAccFilestoreInstance_psc(t *testing.T) {
+	t.Parallel()
+
+	data := map[string]string{
+		"Name": fmt.Sprintf("tf-test-%d", acctest.RandInt(t)),
+		"Location": "us-central1",
+		"Tier": "REGIONAL",
+	}
+	config, err := executeTemplate(pscInstanceConfigTemplate, data)
+	if err != nil {
+		t.Fatalf("Failed to execute config template: %v", err)
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckFilestoreInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_filestore_instance.instance", "networks.0.connect_mode", "PRIVATE_SERVICE_CONNECT"),
+				),
+			},
+			{
+				ResourceName:            "google_filestore_instance.instance",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"zone"},
+			},
+		},
+	})
+}
+
+const pscInstanceConfigTemplate = `
```

Comment:
Could you use `Nprintf` to do similar? Your implementation makes sense, but it is fairly non-standard and we use a very large number of these test templates.

Reply:
Missed `Nprintf`, thanks.